source/client/__tests__/notification/notification.reducer.spec.js

```diff
@@ -1,39 +1,53 @@
-import { DISMISS_NOTIFICATION, SEND_NOTIFICATION } from "../../application/notification/notification.types"
-import { notificationReducer } from "../../application/notification/notification.reducer"
+'use strict';
```

Comment:
Hey -- great, great, great work on this! This is great experience on upgrading something with breaking changes. A lot of this looks good, but you still have to merge in the latest master. For example, this test in notification.reducer.spec.js, is being replaced with stuff that used to be in master but isn't anymore.

Reply:
Okay. I'll get on that :smile:

Reply:
I've done the merge